rosdep/sources.list.d/30-airtonomy.list

```diff
@@ -1,2 +1,3 @@
 # generic
 yaml https://raw.githubusercontent.com/Airtonomy/rosdistro/main/rosdep/base.yaml
+yaml https://raw.githubusercontent.com/ros/rosdistro/main/rosdep/python.yaml
```

Comment:
@RobertLiu-Atnm This is pointing to the ros python file, not the airtonomy file.  I don't know how this would have worked for you in testing?

Reply:
Right, this is typo.  Let me commit my tested code to correct.